pkg/restapi/issuer/operation/operations.go

```diff
@@ -1728,11 +1740,30 @@ func (o *Operation) hanlDIDExStateMsg(msg service.StateMsg) error {
 	return nil
 }
 
-// read credential manifest from profile URL endpoints
-// TODO for now returning empty manifest, TO BE IMPLEMENTED [issue##561 & issue#563]
-func (o *Operation) readCredentialManifest() *cm.CredentialManifest {
+// Read credential manifest issuer detail from persisted profile data and scope from persisted transaction cred scope.
+// cm.Issuer's ID will be used as issuer ID which identifies who the issuer of the credential(s) will be.
+// Credential Manifest Styles represents an Entity Styles object as defined in credential manifest spec.
+// TODO issue#561 Add credential manifest presentation definition
+func (o *Operation) readCredentialManifest(profileData *issuer.ProfileData,
+	txnCredScope string) *cm.CredentialManifest {
+	if cmDescriptor, ok := o.cmOutputDescriptor[txnCredScope]; ok {
+		return &cm.CredentialManifest{
+			ID:      uuid.NewString(),
+			Version: credentialManifestVersion,
+			Issuer: cm.Issuer{
+				ID:     profileData.ID,
```

Comment:
you can not use profile ID here, this is not issuer ID

Reply:
@sudeshrshetty In here when we are creating credential the for the issuer id we are using profile.url https://github.com/trustbloc/adapter/blob/837ffabdd8ed11224679e6091083ace5460900c5/pkg/restapi/issuer/operation/operations.go#L1568 

Does this implies the issuer adapter is creating credential and thats why using profile URL ? Can that be used here as well ??

Reply:
this looks wrong, don't use that.  Profile ID is some UUID generated to co-relate data in adapter it has nothing to do with actual identity of issuer.

Refer this example for credential manifest: https://identity.foundation/credential-manifest/#credential-manifest---all-features-exercised

For let profile creation option accept issuer ID & Style as input arguments as I mentioned [here](https://github.com/trustbloc/adapter/pull/574#discussion_r791111034)

Reply:
@sudeshrshetty  I have added issuerID and cmstyle in profile data as when New issuer onboards to adapter it need to provide issuer ID and manifest style for that specific profile.

Reply:
where it is set?